src/app/fyle/split-expense/split-expense.page.ts

```
@@ -1101,16 +1351,42 @@
+  private setRecentlySplitExpenses(data: { expenses: Partial<Transaction>[] }): void {
+    this.expensesService.splitExpensesData$.next(data);
+  }
+
+  private getRecentlySplitExpenses(): Observable<{ expenses: Partial<Transaction>[] } | null> {
+    return this.expensesService.splitExpensesData$.asObservable();
+  }
+
+  private clearRecentlySplitExpenses(): void {
+    this.expensesService.splitExpensesData$.next(null);
+  }
+  
```

ESLint

[error] Trailing spaces not allowed.

(no-trailing-spaces)